src/components/ProfilePicture.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, BoxProps, Skeleton } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { fetchProfilePicture } from '../api/cristinApi';
+
+interface ProfilePictureProps {
+  id: string;
+  height: string;
+  hasBorder?: boolean;
+  sx?: BoxProps;
+}
+
+export const ProfilePicture = ({ id, height, hasBorder, sx }: ProfilePictureProps) => {
+  const profilePictureQuery = useQuery({
+    queryKey: ['picture', id],
+    queryFn: () => fetchProfilePicture(id),
+    meta: { errorMessage: false },
+    retry: false,
+  });
+
+  const profilePictureString = profilePictureQuery.isSuccess
+    ? `data:image/jpeg;base64,${profilePictureQuery.data.base64Data}`
+    : '';
+
+  return profilePictureQuery.isFetching ? (
+    <Skeleton variant="circular" sx={{ height: height, aspectRatio: '1/1' }} />
+  ) : (
+    <Box
+      component="img"
+      src={profilePictureString}
+      alt="profile-picture"
+      sx={{
+        aspectRatio: '1/1',
+        height: height,
+        borderRadius: '50%',
+        border: hasBorder ? '0.125rem solid black' : 'none',
+        objectFit: 'cover',
+        ...sx,
```

Comment:
merk at en svakhet med dette er at om vi nå bruker denne komponenten slik: `<ProfilePicture height="5rem" sx={{height:"10rem"}}`, så vil sx.height overstyre height, som er litt rart når det er height som er obligatorisk prop 🤔 
men ser ikke noen god løsning på det sånn det er nå, dessverre. 

En ting som kan være verdt å sjekke ut er å legge en `<Box sx={{height, aspectRatio: '1/1'}}>` over både Skeleton og img, og sørge for at skeleton bruker hele høyden og bredden. Da er det bare en plass som kontrollerer høyden på begge elementene